bugbear.py

```diff
@@ -349,6 +351,35 @@ def visit_ExceptHandler(self, node: ast.ExceptHandler):
         return super().generic_visit(node)
 
 
+class ClassInitVisitor(ast.NodeVisitor):
```

Comment:
Might be cleaner to put this inside the `BugBearVisitor`

Reply:
Done, LMK if that looks better